src/cryptonote_protocol/quorumnet.cpp

```diff
@@ -1287,6 +1289,14 @@ std::future<std::pair<cryptonote::blink_result, std::string>> send_blink(void *o
             MINFO("Relaying blink tx to " << as_hex(remotes[i].first) << " @ " << remotes[i].second);
             snw.snn.send(remotes[i].first, "blink", data, send_option::hint{remotes[i].second});
         }
+    } catch (...) {
+        std::unique_lock<std::shared_timed_mutex> lock{pending_blink_result_mutex};
+        auto it = pending_blink_results.find(blink_tag); // Look up again because `brd` might have been deleted
+        if (it != pending_blink_results.end()) {
+            try {
+                promise.set_exception(std::current_exception());
```

Comment:
Is this meant to use `it`? 

Earlier you do
`brd.promise = std::move(promise)`